public/js/monitor.js

```diff
@@ -7,529 +7,524 @@
 /* global ga */
 
 if (typeof TextEncoder === "undefined") {
-  const cryptoScript = document.createElement("script");
-  const scripts = document.getElementsByTagName("script")[0];
-  cryptoScript.src = "/dist/edge.min.js";
-  scripts.parentNode.insertBefore(cryptoScript, scripts);
+    const cryptoScript = document.createElement("script");
+    const scripts = document.getElementsByTagName("script")[0];
+    cryptoScript.src = "/dist/edge.min.js";
+    scripts.parentNode.insertBefore(cryptoScript, scripts);
 }
 
-
 function findAncestor(el, cls) {
-  while ((el = el.parentElement) && !el.classList.contains(cls));
-  return el;
+    // FIXME: no op
```

Comment:
This function is empty.

Reply:
Technically, I think the loop is reassigning the value of `el` on each iteration until it finds a match, so I don't think it's a noop.